src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -117,6 +122,15 @@ public ProjectsListViewController(final Model model, final Controller controller
             searchTextField.setText("");
          }
       });
+      //From https://stackoverflow.com/questions/14965318/javafx-method-selectall-just-works-by-focus-with-keyboard
+      searchTextField.focusedProperty().addListener(observable -> {
```

Comment:
This does not seem to be correct from the javadoc
![image](https://user-images.githubusercontent.com/5772511/214838872-68a3fc16-c55a-4877-8605-4ec4f6bb898e.png)